push2dockerhub.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# 
+# This script is intended to be run in the deploy stage of a travis build
+# It checks to make sure that this is a not a PR, and that we have the secure
+# environment variables available and then checks if this is either the master
+# or develop branch, otherwise we don't push anything
+#
+# NOTE: IMAGE_NAME is expected to be passed in via the environment so that this
+# script can be more general
+#
+# sychan@lbl.gov
+# 8/31/2017
+
+TAG=`if [ "$TRAVIS_BRANCH" == "master" ]; then echo "latest"; else echo $TRAVIS_BRANCH ; fi`
+COMMIT=${TRAVIS_COMMIT:-`git rev-parse --short HEAD`}
```

Comment:
Can you explain this line with a comment for non-bashers? it's pretty opaque.

Reply:
Done